src/command_modules/azure-cli-eventhubs/azure/cli/command_modules/eventhubs/_params.py

```diff
@@ -0,0 +1,119 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import tags_type, get_enum_type, resource_group_name_type, name_type, get_location_type, get_three_state_flag
+from azure.cli.core.commands.validators import get_default_location_from_resource_group
+
+
+# pylint: disable=line-too-long
+def load_arguments_eh(self, _):
+    from knack.arguments import CLIArgumentType
+    from azure.mgmt.eventhub.models.event_hub_management_client_enums import KeyType, AccessRights, EntityStatus, SkuName, SkuTier
+
+    rights_arg_type = CLIArgumentType(options_list=['--rights'], nargs='+', arg_type=get_enum_type(AccessRights), help='Authorization rule rights of type list')
```

Comment:
"...right of type list".  This is ambiguous.  It should call out: "Space-separated list of rule rights."

Reply:
Resolved, appended text 'Space-separated list of'